portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -81,6 +81,27 @@ const addSubmissionDateToIssuedFacilities = async (dealId) => {
   return facilities;
 };
 
+/*
+  If facility has been changed to issued (after first submission)
+  When submitting to UKEF, have to remove the changedToIssued flag
+  Ensures that cannot update this facility anymore
+*/
+const removeChangedToIssued = async (dealId) => {
+  const facilities = await getAllFacilitiesByDealId(dealId);
+
+  facilities.forEach(async (facility) => {
+    const { _id, changedToIssued } = facility;
+
+    if (changedToIssued) {
+      const update = {
+        changedToIssued: false,
+      };
```

Comment:
I'm not familiar with the logic for this flag - do we need to add a check to make sure this *is not* added if it does _not_  change? I.e 

- facility is issued. Will `changedToIssued` then exist? (seem's it shouldn't?)
- facility is unissued then becomes issued, all good.

Reply:
Or looking at the comment for this function, should it perhaps be called `canChangeToIssued` instead?

I'm finding it a bit confusing just looking at the code/comment - if a facility **_has_** changed to issued, we remove the `changedToIssued` flag?

Reply:
changedToIssued only happens if facility changed from unissued to issued once application submitted to UKEF.  When draft, that flag is not touched.  But I need that flag removed once submitted to UKEF.  Really is a temporary flag to allow resubmission and populating an array and change buttons.

Reply:
What do you think about a more generic naming like `canResubmit`? Just it wasn't clear to me what was happening/why

Reply:
I agree with @ttbarnes renaming the flag to more understandable name i.e. `canResumit` will be a great improvement as it could create confusion.

Reply:
changed to canResubmitIssuedFacilities